### PR TITLE
chore(Update): updated to alpha.53. 

### PR DESCRIPTION
**Pull request by NathanWalker**
Also made routerLink default to active color on first load to improve usability.

### CODE REVIEW

File: src/scss/app.scss [hunk 1]
```diff
@@ -5,3 +5,17 @@ html {
     margin: 30px 0 90px;
   }
 }
+
+$color-white: #fff;
+
+a {
```
Comment (ghpabs) [inline src/scss/app.scss]: Target a class instead of the `a` DOM element.

File: src/scss/app.scss [hunk 2]
```diff
@@ -5,3 +5,17 @@ html {
...
+  &.router-link-active {
+    background-color: $btn-primary-bg;
```
Comment (ghpabs) [inline src/scss/app.scss]: Let's use Sass's built-in `@extend` feature.

**Comment by ghpabs** [comment]
Thanks for sending the PR. It's nice to see people contributing back to the project.

**Comment by ghpabs** [comment]
Outdated - Angular 2.0.0-alpha.54 has now been released and also implemented in this project.

Thanks for your help!

**Comment by NathanWalker** [comment]
This PR addressed other things than the update (likely what @Hack-Point was commenting on), but ok, I guess I'll move my efforts elsewhere.

**Comment by ghpabs** [comment]
Can you be a bit more direct to the point? If you're talking about the button active state, then you should have read my comment.

**Pull request closed without merging**